src/ConfigFile.h

```diff
@@ -88,17 +88,20 @@ typedef enum
 
 typedef enum
 {
-    CFG_ROS_USER_LAN_DISABLED = -2,  //sentinel
+    CFG_ROS_USER_LAN_MALFORMED = -3,  //sentinel
+    CFG_ROS_USER_LAN_ALL = -2,  //sentinel
     CFG_ROS_USER_LAN_AUTO = -1,  //sentinel
     CFG_ROS_USER_LAN1 = ROS_USER_LAN1,
     CFG_ROS_USER_LAN2 = ROS_USER_LAN2,
 } Ros_UserLan_Port_Setting;
```

Comment:
I understand why you opted to reuse the `Ros_UserLan_Port_Setting` type, but it does make the parsing code a little harder to read.

Reply:
Yeah I tried doing it both ways, and this had much less duplicated code and was overall cleaner in my opinion. I don't especially like it either though.